connector/src/main/java/org/geysermc/connector/network/ConnectorServerEventHandler.java

```diff
@@ -95,6 +95,17 @@ public BedrockPong onQuery(InetSocketAddress inetSocketAddress) {
             pong.setMaximumPlayerCount(config.getMaxPlayers());
         }
 
+        // The ping will not appear if the MOTD + sub-MOTD is of a certain length.
+        if (pong.getMotd().length() + pong.getSubMotd().length() > 339) {
+            // Split the difference
+            if (pong.getMotd().length() > 170) {
```

Comment:
Could this be added as a note to the config so users know it will be trimmed to 170 characters?